components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -70,6 +85,29 @@ class RhsRootPost extends React.PureComponent {
         };
     }
 
+    handleShortcutEmojiForLastMessage = (isLastPost) => {
+        if (isLastPost) {
+            const {post, enableEmojiPicker, channelIsArchived,
+                actions: {openEmojiPickerForLastMessageFrom}} = this.props;
+
+            // Setting the last message emoji action to empty to clean up the redux state
+            openEmojiPickerForLastMessageFrom(Locations.NO_WHERE);
+
+            const isPostDeleted = post && post.state === Posts.POST_DELETED;
```

Comment:
Minor nit: maybe use `isDeletedPost` for the const name here like you did in post_info.jsx.